### ISSUE TITLE
Problems setting up a Repository: A label (english) is quite incorrect and confusing

### ISSUE THREAD

**Issue by morsedl** [cyborgbackup/cyborgbackup#88]
Hi, as mentioned in a couple of recent posts here, I've struggled with but regardless successfully installed Cyborg Backup (CB) using slightly-adapted versions of the instructions for doing so on the main website (see issues #87 for details).

Related to this, however, is a problem that cost me a bit of time to figure out. One which is critically important, yet fortunately easy to fix:

When setting up a Repository in CB, the (english) prompts for the repo name and path. But then the third bit of information it asks for is the "Repo Key".

This is completely INCORRECT and took me a bit to figure out (as I could see things failing at a rough level within the output from `docker logs ...`, but I could never find more detailed information to point me in the right direction).

The prompt should ask for the "REPO PASSPHRASE", not for "KEY". 

As I'm sure most here are aware, basic Borg uses both a passphrase and repo key. The repo key has a specific format that start with `BORG_KEY ...` and then several lines of data. I spun my wheels a bit trying a direct copy and paste of the repo key into this form field, stripping out the leading `BORG_KEY ` part, doing that and stripping the blank spaces out due to copy-and-paste of a multi-line value into a single-line form field, etc. etc.

Nothing worked until it occurred to me to try using the repo passphrase, at which point most of my problems were resolved and I could move forward.

Unfortunately I then encountered the problems with detailed Catalog retrieval, which in turn prevents restore testing, etc. as described in issue #79. I hope to help get that resolved soon, as I consider CB the best UI for Borg I've seen. And I like it, because, excepting Django, it uses many of the (heavily Python) libraries I myself would use to construct such a system (I would have been more include to use Pyramid instead of Django as a basis, in order to have a lighter and more-tailored basis).

In summary, I think technically speaking, CB is very good, at least as far as I've seen, and once things were (mostly) working, things ran smoothly, clearly, and efficiently. Overall there the documentation is too sparse, inaccurate at times, and hard to read (the author I presume is non-native English speaker), yet fundamental sound for the most part in clarifying "what's most important" to know re: setting up and running CB.

So, I'd very much like to help get some of these issues resolved, in particular the problems identified in issue #79. I'll follow up shortly there. I just wanted to note here that I am happy to help that I am also happy to help update the online documentation (#87) and help correct label(s) (this issue; I'm native English and fluent German speaker).
